dissect/target/loaders/hyperv.py

```diff
@@ -0,0 +1,160 @@
+from __future__ import annotations
+
+import base64
+from pathlib import Path
+from typing import TYPE_CHECKING, Optional
+from xml.etree.ElementTree import Element
+
+from defusedxml import ElementTree
+from dissect.hypervisor import hyperv
+
+from dissect.target import container
+from dissect.target.helpers import fsutil
+from dissect.target.loader import Loader
+
+if TYPE_CHECKING:
+    from dissect.target.target import Target
+
+DRIVE_CONTROLLER_GUIDS = [
+    # Microsoft Emulated IDE Controller
+    "83f8638b-8dca-4152-9eda-2ca8b33039b4",
+    # Synthetic SCSI Controller
+    "d422512d-2bf2-4752-809d-7b82b5fcb1b4",
+]
+
+
+def is_hyperv_xml(path: Path) -> bool:
+    """Return if an XML file is a valid Hyper-V configuration.
+
+    Args:
+        path: Path to the file to check.
+    """
+    if path.suffix != ".xml":
+        return False
+
+    try:
+        et = ElementTree.fromstring(path.read_bytes())
+        element_names = [el.tag for el in et]
+        return et.tag == "configuration" and "manifest" in element_names
```

Comment:
The root element is `configuration`, so it'll never how up in `element_names`, since that will consist of all child element of the `configuration` element.